nss-cohort-37/Bangazon-Site-Green-Monkeys
# Mac add to cart

# Description

Users can now browse a list of products and add items to their cart from both the item list page, and the item detail page.



## Type of change

Please delete options that are not relevant.


# Testing Instructions

- view the list of products
- click `add to cart` 
- confirm and add to cart
- view the cart and confirm the item was added to the cart


# Checklist:
